.license_scout.yml

```diff
@@ -75,6 +75,9 @@ fallbacks:
     - name: epgsql
       license_id: BSD-3-Clause
       license_content: https://github.com/chef/epgsql-1/blob/master/LICENSE
+    - name: erlcloud
```

Comment:
Since this is adding the license_scout dep, it might be useful to cherrypick the change adding erlcloud as a dep along with this. This will also be a good to merge in before all of the sig_v4 related work is merged.

Reply:
specifically: https://github.com/chef/chef-server/pull/2094/commits/d532550c777f8d521f556baa3bfa2045abbc076b

Reply:
It looks like that commit also changes pointers for mini_s3, but I can split up the commit and just cherrypick erlcloud.

Reply:
Done.